src/main/java/conversion/BaseServlet.java

```diff
@@ -417,6 +414,25 @@ private String getFileName(final Part part) {
         return null;
     }
 
+    /**
+     * Checks if the callbackUrl parameter was included in the request, if so it
+     * will queue the callback into the callbackQueue.
+     *
+     * @param jsonData the jsonData that is sent to the URL
+     * @param params the request parameters
+     */
+    private void handleCallback(final Individual individual, final Map<String, String[]> params) {
+        final String[] rawParam = params.get("callbackUrl");
+
+        if (rawParam != null) {
+            final String callbackUrl = String.join(" ", rawParam);
```

Comment:
I think it would be better to use callBackUrl = rawParam[0] here
String.join() may look confusing to others as it suggests there are multiple parts to the URL param

Reply:
Have changed accordingly